milestone2.ts

```diff
@@ -92,23 +98,23 @@ let findAverage: GetNumber; // Define lambda function here
   Return -1 if no one is from San Francisco.
   Hint: Use a lambda function as a value
 */
-let findIndexAns: GetNumber; // Code here
+let findIndexAns: GetNumber = (data: Volunteer[]): number => data.findIndex(vol => vol.city === 'SF'); // findIndexAns function
 
 //console.log(findIndexAns(userData));
 
 // Question 3: Filtering data
 /* Use data handling function(s) to find all of the volunteers from California (CA) over an age threshold n
  */
 type GetVolunteers = (data: Volunteer[], minAge: number) => Volunteer[];
-let findCAOverN: GetVolunteers; // Code here
+let findCAOverN: GetVolunteers = (data: Volunteer[], minAge: number): Volunteer[] => data.filter(vol => vol.state === 'CA' && vol.age >= minAge); // findCAOverN function
```

Comment:
Double defining all of the types again, this can make the start of your functions harder to read for your peers/reviewers.